internal/sirius/get_deputy_events.go

```diff
@@ -0,0 +1,108 @@
+package sirius
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+type DeputyEvents []DeputyEvent
+
+
+type User struct {
+	UserId int `json:"id"`
+	UserDisplayName string `json:"displayName"`
+	UserPhoneNumber string `json:"phoneNumber"`
+}
+
+type Event struct {
+	OrderType        string         `json:"orderType"`
+	SiriusId         string         `json:"orderUid"`
+	OrderNumber      string         `json:"orderCourtRef"`
+	DeputyID         string         `json:"personId"`
+	DeputyName       string         `json:"personName"`
+	OrganisationName string         `json:"organisationName"`
+	Changes          []Changes      `json:"changes"`
+	Client           []ClientPerson `json:"additionalPersons"`
+}
+
+type Changes struct {
+	FieldName string `json:"fieldName"`
+	OldValue  string `json:"oldValue"`
+	NewValue  string `json:"newValue"`
+}
+
+type ClientPerson struct {
+	ClientName     string `json:"personName"`
+	ClientId       string `json:"personId"`
+	ClientUid      string `json:"personUid"`
+	ClientCourtRef string `json:"personCourtRef"`
+}
+
+type DeputyEvent struct {
+	TimelineEventId  int    `json:"id"`
+	Timestamp        string `json:"timestamp"`
+	EventType        string `json:"eventType"`
+	User             User   `json:"user"`
+	Event            Event  `json:"event"`
+}
+
+func (c *Client) GetDeputyEvents(ctx Context, deputyId int) (DeputyEvents, error) {
+	var v DeputyEvents
+
+	req, err := c.newRequest(ctx, http.MethodGet, fmt.Sprintf("/api/v1/timeline/%d", deputyId), nil)
+
+	if err != nil {
+		return v, err
+	}
+
+	resp, err := c.http.Do(req)
+	if err != nil {
+		return v, err
+	}
+
+	defer resp.Body.Close()
+
+	if resp.StatusCode == http.StatusUnauthorized {
+		return v, ErrUnauthorized
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return v, newStatusError(resp)
+	}
+	err = json.NewDecoder(resp.Body).Decode(&v)
+
+	DeputyEvents := EditDeputyEvents(v)
```

Comment:
I think it's a bit confusing to name a variable the same as a type (line 10), maybe this should be `deputyEvents`?

Reply:
Another option is to consider naming the type to DeputyEventCollection.

Reply:
Variables should be lower case. 

I would have prefered the type to be `DeputyEvents` as the plural already impliies a collection. Not convinced the type is necessary either, seeing as it is just aliasing an array. Both are fine though.